react-state/my-app/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import logo from './logo.svg';
+import './App.css';
+import ToggleButton from './ToggleButton';
+
+function App() {
+  return (
+    <div className="App">
```

Comment:
You can just remove all this CRA stuff and have this be just the new exercise code.